documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -510,6 +510,7 @@
                   "optimizing_team_collections",
                   "healthy_repartitioning",
                   "healthy_removing_server",
+                  "team_healthy",
```

Comment:
By convention, we use `healthy_xxx`. 
Do you mean `healthy_team`?

Reply:
Yes.